src/main/flight/imu.c

```diff
@@ -365,61 +365,58 @@ static bool imuIsAccelerometerHealthy(float *accAverage)
 //   - reset the gain back to the standard setting
 static float imuCalcKpGain(timeUs_t currentTimeUs, bool useAcc, float *gyroAverage)
 {
-    static bool lastArmState = false;
-    static timeUs_t gyroQuietPeriodTimeEnd = 0;
-    static timeUs_t attitudeResetTimeEnd = 0;
-    static bool attitudeResetCompleted = false;
-    float ret;
-    bool attitudeResetActive = false;
+    static enum {
+        stArmed,
+        stRestart,
+        stQuiet,
+        stReset,
+        stDisarmed
+    } arState = stDisarmed;
```

Comment:
does `arState` mean `attitudeResetState` ?

Reply:
Yes. I don't like too long names when variable score is so small.. Maybe only 'state' will be less confusing